pkg/vere/io/mesa.c

```diff
@@ -3064,7 +3007,12 @@ _mesa_io_talk(u3_auto* car_u)
   }
 
   u3_Host.wax_u.data = sam_u;
-  uv_udp_recv_start(&u3_Host.wax_u, _ames_alloc, _mesa_recv_cb);
+  uv_udp_recv_start(&u3_Host.wax_u, _mesa_alloc, _mesa_recv_cb);
+
+  c3_i rec_i = 2 * 1024 * 1024;
+  uv_recv_buffer_size((uv_handle_t*)&u3_Host.wax_u, &rec_i);
+
+  uv_send_buffer_size((uv_handle_t*)&u3_Host.wax_u, &rec_i);
```

Comment:
It seems like we should read the buffer sizes (setting `rec_i = 0`) before setting them, to make sure we're not shrinking them.